src/unix/lwt_unix.cppo.ml

```diff
@@ -1149,6 +1149,14 @@ let chdir name =
   else
     run_job (Jobs.chdir_job name)
 
+external getcwd_job : unit -> string job = "lwt_unix_getcwd_job"
+
+let getcwd () =
+  if Sys.win32 then
+    Lwt.return (Unix.getcwd ())
```

Comment:
That's inspired by the other functions in the module. I don't think it's optimal. It'll probably get better if #328 gets resolved?